backend/src/apiserver/data/context/ranking.py

```diff
@@ -136,6 +139,50 @@ async def add_new_training(dsrc: Source, new_event: NewTrainingEvent) -> None:
         )
 
 
+async def add_new_training(dsrc: Source, new_event: NewTrainingEvent) -> None:
+    async with get_conn(dsrc) as conn:
+        event_ids = await add_training_event(
+            conn,
+            1,
```

Comment:
Deze 1 is puur als test toch? Want klassement id kan anders zijn

Reply:
ah dit had ik nu als 1 gehardcode omdat het over trainingen toevoegen gaat. Moet dit veranderd worden of is dit iets voor later?

Reply:
Trainingsklassement wordt ook om de zoveel tijd opnieuw aangemaakt, dus ind it geval wel

Reply:
heb dit nu aangepast!